src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -286,5 +290,18 @@ public boolean equals(Object other) {
                     && getInterviewStatus().equals(e.getInterviewStatus())
                     && getAvailability().equals(e.getAvailability());
         }
+
+
+        /**
+         * Allows the {@code InterviewStatus} to be triggered by ApplicationStatus.
+         */
+        public void triggerInterviewStatus(ApplicationStatus applicationStatus) {
+            if (getApplicationStatus().isPresent()) {
+                if (applicationStatus.toString().equals(ACCEPTED_STATUS)
+                        || applicationStatus.toString().equals(REJECTED_STATUS)) {
+                    setInterviewStatus(new InterviewStatus(COMPLETED));
+                }
+            }
```

Comment:
Yup! Thanks for making the change @tiewweijian! LGTM overall.

Will approve and merge now! 💯